### ISSUE TITLE
SketchZoomAsync with Compose Multiplatform 1.7.0-alpha01 does not load any image(s)

### ISSUE THREAD

**Issue by kasem-sm** [panpf/zoomimage#30]
**Describe the bug**
This was printed in my log - `SubsamplingState. resetTileDecoder:setImageSource. error, java.util.concurrent.CancellationException: cleanTileDecoder:resetTileDecoder:contentSizeChanged. contentSize: 1026x2051, 'https://image.png'`

**Affected platforms**

Select of the platforms below -  Android
[Not tested on other platform]

**Affected components**

Select of the components below:
- SketchZoomImageView

**Versions**

- zoomimage version*: v1.1.0-alpha02
- Kotlin version*: 
- Compose version(s)* (Multiplatform): 1.7.0-alpha01
- JDK version(Only Desktop): N/A

**Running Devices**

Please accurately describe the device Model, OS version, and CPU Architecture

- Pixel 7a, Android 14

**Sample code**

```
SketchZoomAsyncImage(
    request = request,
    contentDescription = contentDescription,
    modifier = modifier,
    contentScale = ContentScale.Crop,
    state = imageState,
)
```

**Reproduction steps**

Just run the above code

**Expected behavior**

Should display the image

**Screenshots**

N/A

**Comment by kasem-sm** [comment]
Works perfectly with 1.6.* (CMP)

**Comment by panpf** [comment]
In this case, I suggest waiting for 1.7.0-beta01 and trying again, because alpha means unstable.

**Comment by kasem-sm** [comment]
I can understand. I have to use the alpha as it fixes an important issue for me. Anyway, thank You!

**Comment by panpf** [comment]
Since you have a reason to use it, I will try to solve it.

**Comment by kasem-sm** [comment]
Thank You ☺️. Just for info: Coil3 Works on the alpha v.

**Comment by kasem-sm** [comment]
Interestingly, the images loaded successfully when I tested it in the emulator with API level 28.

**Comment by panpf** [comment]
Can you post the error log?

**Comment by kasem-sm** [comment]
There are no error messages because the image load state shows Success but the painter is null (with CoilZoomAsyncImage)

**Comment by panpf** [comment]
Yes, I need

**Comment by kasem-sm** [comment]
I didn't expect this. I created a separate sample with CMP version 1.7.0-alpha01, and the images did load. After some debugging, I found the issue. I use Haze library, and experimentally tried to remove the Haze modifier from the SketchZoomImage, the image successfully loaded. [Haze](https://github.com/chrisbanes/haze) library recently had a rewrite and now uses the new GraphicsLayer APIs from Compose 1.7 alpha.

As the issue does not seems to be in the ZoomImage lib, I'm closing this. Thank You!

CC: @chrisbanes
